Add encryptSecrets and decryptSecrets route in adminController (#1439)
kuzzleio/kuzzle: doc/1/api/controllers/admin/decrypt-secrets/index.md

---
code: true
type: page
title: decryptSecrets
---

# decryptSecrets

<SinceBadge version="1.10.0" />

> Decrypt an object using [Vault](/core/2/guides/essentials/secrets-vault/).

Reviewer: ```suggestion
Decrypts an object using [Vault](/core/2/guides/essentials/secrets-vault/).
```

> Decrypts an object using [Vault](/core/2/guides/essentials/secrets-vault/).

Reviewer 2: ```suggestion
Decrypts an object using [Vault](/core/1/guides/essentials/secrets-vault/).
```

> Decrypts an object using [Vault](/core/1/guides/essentials/secrets-vault/).

Reviewer 3: ```suggestion
Decrypts keys stored in the [Vault](/core/1/guides/essentials/secrets-vault/).
```
